source/_integrations/stream.markdown

```diff
@@ -13,48 +13,17 @@ ha_codeowners:
 ha_domain: stream
 ---
 
-The `stream` integration provides a way to proxy live streams through Home Assistant. The integration currently only supports proxying H.264 source streams to the HLS format and requires at least FFmpeg >= 4.
+The `stream` integration provides a way to proxy live streams through Home Assistant as an internal component used by the [`camera`](/integrations/camera) integration. The integration currently only supports proxying H.264 source streams to the HLS format and requires at least FFmpeg >= 4.
 
 ## Configuration
 
 To enable this component, add the following lines to your `configuration.yaml` file:
```

Comment:
The `stream` component is automatically loaded and enabled by the `camera` platforms that support it. No separate `configuration.yaml` entry is necessary.